testnet/stacks-node/src/run_loop/helium.rs

```diff
@@ -24,13 +28,16 @@ impl RunLoop {
         config: Config,
         boot_exec: Box<dyn FnOnce(&mut ClarityTx) -> ()>,
     ) -> Self {
+        let (attachments_tx, attachments_rx) = sync_channel(1);
```

Comment:
Only room for one outstanding inter-thread attachment request?  You might want to increase this to something like 1024.

Reply:
Ah it's not one attachment at a given time, it's one batch (with all the attachments from a given block) at a given time, which should be fine.